projects/adrv9009/zc706/Makefile

```diff
@@ -5,30 +5,36 @@
 
 PROJECT_NAME := adrv9009_zc706
 
+M_DEPS += add_files-norecurse$ad_hdl_dir/library/common/util_pulse_gen.v
```

Comment:
This is an issue, should probably fix generator script

Reply:
Can you fix it?

Reply:
Never mind, I will handle it. Thanks for the feedback.